modular_skyrat/modules/neural-lacing/code/delacer.dm

```diff
@@ -0,0 +1,52 @@
+/obj/item/stackremover
+	name = "cortical stack remover"
+	desc = "A pair of tweezer-like knives, used to quickly extract a stack after death."
+	icon = 'icons/obj/surgery.dmi'
+	icon_state = "shears"
+	flags_1 = CONDUCT_1
+	item_flags = SURGICAL_TOOL
+	toolspeed  = 1
+	force = 12
+	w_class = WEIGHT_CLASS_NORMAL
+	throwforce = 6
+	throw_speed = 2
+	throw_range = 5
+	custom_materials = list(/datum/material/iron=8000, /datum/material/titanium=6000)
+	attack_verb_continuous = list("shears", "snips")
+	attack_verb_simple = list("shear", "snip")
+	sharpness = SHARP_EDGED
+	custom_premium_price = PAYCHECK_MEDIUM * 14
+
+/obj/item/stackremover/syndicate
+	name = "blood-red cortical stack remover"
+	desc = "A pair of tweezer-like knives, used to quickly extract a stack after death. This one is shaped like a double-helix; with blood-red blades. Are you a believer?"
+	toolspeed  = 2
```

Comment:
Is this intended to be slower? Seems odd.

Reply:
Nope.